sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/red_team/_red_team.py

```diff
@@ -207,30 +207,30 @@ async def _log_redteam_results_to_mlflow(
         self,
         redteam_output: RedTeamResult,
         eval_run: EvalRun,
-        data_only: bool = False,
+        _skip_evals: bool = False,
     ) -> Optional[str]:
         """Log the Red Team Agent results to MLFlow.
         
         :param redteam_output: The output from the red team agent evaluation
         :type redteam_output: ~azure.ai.evaluation.RedTeamOutput
         :param eval_run: The MLFlow run object
         :type eval_run: ~azure.ai.evaluation._evaluate._eval_run.EvalRun
-        :param data_only: Whether to log only data without evaluation results
-        :type data_only: bool
+        :param _skip_evals: Whether to log only data without evaluation results
+        :type _skip_evals: bool
```

Comment:
Nit: do we expect external customers to use this flag? If not I suggest we hide it in `kwargs` and not having ref doc for it.